lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -287,7 +299,7 @@ class EngineAutofillForm {
             element.onInput.listen((html.Event e) {
               if (items![key] == null) {
                 throw StateError(
-                    'Autofill would not work withuot Autofill value set');
+                    'Autofill would not work without Autofill value set');
```

Comment:
Is this an error that could happen due to a framework-side bug? If so, can the error message be more specific? For example, would it help if it mentioned the `key` that was missing and why it was expected?